modules/admin_manual/pages/installation/letsencrypt/using_letsencrypt.adoc

```diff
@@ -304,7 +304,7 @@ which can be run as follows:
 
 [source,console]
 ----
-sudo /etc/letsencrypt/list.sh
+sudo sh /etc/letsencrypt/list.sh
```

Comment:
The `sh` won't hurt, but this should run fine without `sh` - what goes wrong? Why do we need to add `sh` here?

Reply:
sh is not needed, also see the other PR for the same thing

Reply:
@ChrisEdS please respond and let us know why you think that `sh` is needed here.

Reply:
I tested during a deployment and I had to add the `sh`. Maybe I did something wrong.

Reply:
I can prove you that `sh` is not necessary. I am using LE sucessfully including scripts I created for myself.